### PR TITLE
chore: combine and sort imports

### PR DESCRIPTION
**Pull request by jjaakola-aiven**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -135,6 +135,8 @@ extend-ignore = [
 ]
 
 [tool.ruff.isort]
+combine-as-imports = true
+lines-between-types = 1
```
Comment (jjaakola-aiven) [inline pyproject.toml:139]: This is the change.

File: pyproject.toml [hunk 2]
```diff
@@ -135,6 +135,8 @@ extend-ignore = [
 ]
 
 [tool.ruff.isort]
+combine-as-imports = true
```
Comment (aiven-anton) [inline pyproject.toml:138]:
```suggestion
```

This seems to have no effect?

**Review by aiven-anton — APPROVED** [review]
Thanks, this looks good, I'll merge once we've sorted the pipeline.

**Comment by aiven-anton** [comment]
Confirmed this is failing with same set of tests as main.

**Pull request merged**